Lib/gftools/fix.py

```diff
@@ -609,24 +609,33 @@ def _add_blank_glyph_to_gid1(ttfont):
     from nanoemoji.reorder_glyphs import reorder_glyphs
     from nanoemoji.util import load_fully
     from fontTools.ttLib.tables._g_l_y_f import Glyph
+    from fontTools.ttLib.tables.otTables import NO_VARIATION_INDEX
     ttfont = load_fully(ttfont)
     glyph_order = ttfont.getGlyphOrder()
     glyf_table = ttfont["glyf"]
 
+    if len(glyph_order) < 2:
+        return
     gid1 = glyph_order[1]
     if glyf_table[gid1].numberOfContours == 0:
         return
     hmtx = ttfont["hmtx"]
     blank_glyph = Glyph()
-    blank_name = "blank"
+    blank_name = ".null" if ".null" not in glyph_order else "emptyglyph"
```

Comment:
you may want to `assert "emptyglyph" is not in glyph_order` maybe just to be on the safe side.